### PR TITLE
timeformat for add meetup changed

### PR DESCRIPTION
**Pull request by exploreshaifali**
Fix issue #143

### CODE REVIEW

File: systers_portal/templates/meetup/add_meetup.html [hunk 1]
```diff
@@ -39,9 +39,9 @@
   })
 
   $('.timepicker').pickatime({
-    format: 'HH:i',
-    formatSubmit: 'HH:i',
+    format: 'H:i',
+    formatLabel: 'h:ia',
```
Comment (ana-balica) [inline systers_portal/templates/meetup/add_meetup.html]: The accepted formats for TimeField are https://docs.djangoproject.com/en/1.8/ref/forms/fields/#timefield
Why would you switch to 12 hour format and day time period?

Comment (exploreshaifali) [inline systers_portal/templates/meetup/add_meetup.html]: Because when I display time `{{ meetup.time }}` it appears in 12 hours formate with day time period.

Comment (ana-balica) [inline systers_portal/templates/meetup/add_meetup.html]: Ah, it's format label that changed. Ok. Why did you remove formatSubmit at all?
Also you can adjust the display of date and time in template to whichever format you want - https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#date

Comment (exploreshaifali) [inline systers_portal/templates/meetup/add_meetup.html]: Thanks! Display format changed.

**Comment by Gayathrimenakath** [comment]
I think the fix is correct, but something is wrong with travis CI. It seems like it is not setting the SECRET_KEY from the details from the above Travis CI error details.

**Comment by prachiagrawal269** [comment]
Is this issue still open to work upon?
